src/components/ImageMetadata/index.tsx

```diff
@@ -69,7 +70,7 @@ const ImageMetadata: React.FC<ImageMetadataProps> = ({
         )
       }
       {
-        caption && `${caption} `
+        caption && <span dangerouslySetInnerHTML={{ __html: `${caption} ` }} />
```

Comment:
small nit. is the string literal necessary? seems to be working for me without it

```suggestion
        caption && <span dangerouslySetInnerHTML={{ __html: caption }} />
```

Reply:
The string literal is necessary due to the space being needed between the subtitle and the attribution. This is why the string literal was used before. As you can see above line 72 had a space in the string literal as the last character.

Reply:
if no caption (falsy), then no span here, right?

Reply:
oh I didn't notice the space. can you add a code comment to this to clarify